sig-apps/Charter.md

```diff
@@ -0,0 +1,131 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers developing, deploying, and operating applications on Kubernetes with a focus on the application developer and application operator experience.
+
+### Goals
+
+* Discussion of how to define, develop, and run applications in Kubernetes. This is in order better understand needs, improve on Kubernetes solutions, and share lessons to grow the number of applications running on Kubernetes
+* Foster an ecosystem of tools to aid in developing, deploying, and operating applications on Kubernetes
+* Be the voice of the people running applications in Kubernetes. This includes providing input on Kuberentes features, suggesting new features, and sharing the perspective of application developers and operators in the development of Kubernetes
+* Enable varying workloads (e.g., databases, web services, machine learning) to run on Kubernetes through development of the Workloads API
+* Develop subprojects that foster interoperability between tools used to develop, deploy, and operate applications
+* Create documentation to enable application developers, application operators, and supporting tool developers to leverage Kubernetes
+* Develop and maintain Kubernetes subprojects that aid in the development, deployment, and operation of applications on Kubernetes
+
+### Non-goals
+
+* Do not endorse one particular ecosystem tool
+* Do not pick which apps to run on top of Kubernetes
+* Do not recommend one way to do things (e.g., picking a template language)
+
+## Roles
+
+Membership for roles tracked in: [sigs.yaml]
+
+- Chair
+  - Run operations and processes governing the SIG
+  - Seed members established at SIG founding
+  - Chairs *MAY* decide to step down at anytime and propose a replacement.  Use lazy consensus amongst
+    chairs with fallback on majority vote to accept proposal.  This *SHOULD* be supported by a majority of
+    SIG Members.
+  - Chairs *MAY* select additional chairs through a [super-majority] vote amongst chairs.  This
+    *SHOULD* be supported by a majority of SIG Members.
+  - Chairs *MUST* remain active in the role and are automatically removed from the position if they are
+    unresponsive for > 3 months and *MAY* be removed if not proactively working with other chairs to fulfill
+    responsibilities.
+  - Number: 2-3
+  - Defined in [sigs.yaml]
+
+- Subproject Owners
+  - Scoped to a subproject defined in [sigs.yaml]
+  - Seed members established at subproject founding
+  - *MUST* be an escalation point for technical discussions and decisions in the subproject
```

Comment:
Given that the subproject has no higher technical appeal within the sig, I would capture here that the sig prefers technical decisions to remain within the subproject.

Reply:
The only one i'm concerned about are workload controllers - I'd like to see some detail about that subproject called out since that today owns a critical part of k/k code.

Reply:
@smarterclayton The workload controllers do have one higher level of appear and that's via SIG Architecture. It's not for everything but is for the things SIG Architecture owns. Same applies to everything else in core.

What other sorts of details are you looking to see called out? Since I'm not sure what you're looking for I'm not sure how to craft it.

Reply:
Typically the SIG is point of contact for cross cutting issues such as - giving community updates, ensuring tests are passing before release, triaging release blocking issues, writing release notes, updating documentation, etc.  Should these responsibilities fall on the workloads Controllers subproject directly?  Does the WL Controllers subproject directly assume all responsibilities for being in kubernetes/kubernetes?

Reply:
@pwittrock Because SIG Apps doesn't have technical leads the owners of the code are the technical leads for their areas. Since we have a few concerns this lets people who own different concerns make technical decisions as appropriate fore there.

Release notes, community updates, and some of these other things have fallen on SIG Apps rather than a sub-project in the past.